packages/datagateway-common/src/card/cardView.component.tsx

```diff
@@ -760,7 +760,7 @@ const CardView = (props: CardViewProps): React.ReactElement => {
                 })}
               </List>
             ) : (
-              <Grid xs={12} md={8}>
+              <Grid item xs={12} md={8}>
```

Comment:
I assume `item` is always going to be true, else this code will not work. Safety alternative is `item={item}`

Reply:
`item` is a boolean prop for the MUI `Grid` component which defaults to false. This is shorthand for `item={True}`, you can see the same thing elsewhere with the `container` prop.

https://material-ui.com/api/grid/#props

Reply:
I understand the shorthand, just needed to know it would be true in all circumstances. I'll give it a tick.